src/engraving/libmscore/property.h

```diff
@@ -392,6 +392,9 @@ enum class Pid {
     TEMPO_CHANGE_TYPE,
     TEMPO_EASING_METHOD,
 
+    PAN,
```

Comment:
There's a mismatch between the place where these 2 lines got added and the corresponding lines in properties.cpp.
Moving these 2 here to above `TEMPO_CHANG_TYPE or(mo moving the corresponding lines in proerties.cpp down) might resolve the crash in the unit test

Reply:
And indeed it does ;-)